packages/react-renderer-demo/src/app/src/components/navigation/examples-definitions.js

```diff
@@ -7,7 +7,7 @@ import SelectText from './examples-texts/select';
 
 export const baseExamples = [{
   component: componentTypes.TEXT_FIELD,
-  link: `?component=${componentTypes.TEXT_FIELD}`,
+  link: componentTypes.TEXT_FIELD,
```

Comment:
links and components are now the same, we should use only one

Reply:
Yeah. Links and component are a bit different. I would like them to be separate. It was like that before the srr switch and it was really confusing.